tests/array_map.phpt

```diff
@@ -1,7 +1,7 @@
 --TEST--
 Test with internal callbacks
 --SKIPIF--
-<?php if (!version_compare(phpversion(), "5.5", '<')) echo "skip < PHP 5.5 needed\n"; ?>
+<?php if (!version_compare(phpversion(), "5.5", '>=')) echo "skip >= PHP 5.5 needed\n"; ?>
```

Comment:
No. I thought I had removed these all. Could tiu check if there are more of these?